toolkit/templates/instruction-list.html

```diff
@@ -64,6 +64,24 @@ <h2 class="instruction-list-item-body">{{ item.body }}</h2>
             {% endwith %}
           {% endfor %}
         {% endif %}
+
+        {% if item.content %}
```

Comment:
Having `item.content` and `item.body` is quite confusing, they both imply the same thing. It would be preferable if `item.body` was called something else I think. Header or something. Although that's going to be a breaking change.

Reply:
item.body is the heading - the whole template itself has misleading property names. We should think about refactoring.

Reply:
How much work would it be to change? If we made the breaking change here, it'd only be the buyer-frontend that needs fixing now. Other apps can be updated as they pull in the change.

Reply:
(If we did do something like that, which sounds a good idea to me, then we'd need to make sure the failure mode was super-clear. As in, we should make sure that the functional tests _do_ break, so that it wouldn't be possible to upgrade without fixing by mistake. If for some reason that wasn't straightforward then we could always rename the macro :) )

Reply:
I feel like that specific change would need to go through the tech debt process and is out of scope for this particular issue.

Reply:
I dunno, it feels like we're adding the `content` attribute of the item here which is causing the confusion, so this might be the time to look at it. I've just had a look and this is the only place in the buyer frontend that is using an instruction-list, so should be pretty easy to fix.